docs/extensions-plugins/visual-studio.mdx

```diff
@@ -39,6 +40,9 @@ You can install the Visual Studio Extension through the <a target="blank" href={
 
 Once you've installed the extension, you'll have to restart Visual Studio before using Pieces.
 
+### Showing Pieces Tool Windows
```

Comment:
@robert-at-pieces please take a look here when you have a chance and review for accuracy & completeness 👍🏼